global_variables

```diff
@@ -1,5 +1,5 @@
 IMAGE="registry.fedoraproject.org/fedora:28"
-PKI_VERSION=10.6
-COPR_REPO="@pki/${PKI_VERSION}-nightly"
+COPR_REPO="@pki/10.6"
+TEST_COPR_REPO="@pki/10.6-nightly"
```

Comment:
How about changing it to: `TEST_COPR_REPO="${COPR_REPO}-nightly"` ?

We usually follow (and should follow) `@pki/<PKI_VERSION>` for official COPR repo and `@pki/<PKI_VERSION>-nightly`. I prefer keeping the verison changes in a cfg file as minimum. Your thoughts?

Reply:
Fixed.